test/integration_test.go

```diff
@@ -1057,26 +1057,32 @@ func TestTerragruntOutputFromDependency(t *testing.T) {
 }
 
 func TestTerragruntInputsFromDependency(t *testing.T) {
-	t.Parallel()
+	// NOTE: We can't run this test in parallel because there are other tests that also call `config.ClearOutputCache()` to clear global variable
+	// t.Parallel()
 
```

Comment:
I think this test should be moved in `integration_serial_test.go` if parallel execution is not supported